README.md

```diff
@@ -145,6 +145,9 @@ style is that `Cloudant` (upper-case) is the **package** you load; whereas
 `cloudant` (lower-case) is your **connection** to your database (i.e. the result of
 calling `Cloudant()`).
 
+ It is important to define one _connection_ **only once** during the application lifetime.
```

Comment:
Changed in 00eccec

Reply:
I had tried to escape formatting to make the new content easier to copy/paste, but it looks like the escaping backticks made it through into the commit 00eccec - that will need correcting so that \`cloudant\` becomes correctly marked up as `cloudant`.
Sorry for the confusion there.

Reply:
Corrected in 89d3fb0eb29f3b36000ddc34844c3eb8757cf640